CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByNodeAttributeAction.java

```diff
@@ -32,7 +32,9 @@
  * @author canis_majoris
  */
 @ActionID(category = "Arrange", id = "au.gov.asd.tac.constellation.plugins.arrangements.group.ArrangeByNodeAttributeAction")
-@ActionRegistration(displayName = "#CTL_ArrangeByNodeAttributeAction", surviveFocusChange = true)
+@ActionRegistration(displayName = "#CTL_ArrangeByNodeAttributeAction",
+        iconBase = "au/gov/asd/tac/constellation/plugins/arrangements/group/resources/arrangeByNodes.png",
```

Comment:
arrangeByNode - singlular?

Reply:
Yeah, that should be arrangeByNodes I think. I'll change that

Reply:
@formalhaut69 just letting you know I haven't merged this PR as I was waiting on this change. No rush, whenever you can. Cheers.